modules/021-cni-cilium/openapi/config-values.yaml

```diff
@@ -22,6 +22,14 @@ properties:
       Create routes to Pods on other Nodes.
 
       All Nodes must be located in the same L2 domain.
+  masqueradeMode:
+    type: string
+    enum: ["Netfilter", "BPF"]
+    default: "BPF"
+    description: |
+      Cilium masquerade work mode.
```

Comment:
I'd like to know in the description when do I need to set this field manually instead of relying on the default and what is the impact of changing it (probably it can cause network outage).

Could you please add more detailed description?